README.md

```
@@ -1,13 +1,42 @@
-* 자동 수정 및 PR 생성: 깨진 링크를 수정하고, 변경 사항을 반영한 Pull Request를 자동으로 생성합니다.
+* URL 추출 및 유효성 검사: 레포지토리에서 모든 링크를 추출하고, 404 Not Found 등의 오류를 확인합니다.
+* 대체 링크 탐색 및 수정: 깨진 링크에 대한 대체 가능한 올바른 링크를 찾아 변환합니다.
+* Pull Request 생성: 변경 사항을 반영한 Pull Request를 생성합니다.
+
+## Getting Started
+
+[queensac[bot]](https://github.com/apps/queensac)을 사용하고자 하는 GitHub 레포지토리에 설치합니다.
+
+GitHub Workflow에 아래의 내용을 추가합니다.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)